### PR TITLE
update `ru` locale

### PR DESCRIPTION
**Pull request by chexex**
Updated `ru` locale

### CODE REVIEW

File: django_extensions/locale/ru/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -19,62 +19,101 @@ msgstr ""
...
 #, python-format
 msgid "String (up to %(max_length)s)"
 msgstr "Строка (Не длиннее: %(max_length)s)"
 
-#: templates/django_extensions/widgets/foreignkey_searchinput.html:4
-msgid "Lookup"
-msgstr "Поиск"
+#: validators.py:14
+msgid "Control Characters like new lines or tabs are not allowed."
+msgstr "Символы новой строки и табов недопустимы."
```
Comment (pulysak) [inline django_extensions/locale/ru/LC_MESSAGES/django.po]: Управляющие символы, такие как ....

File: django_extensions/locale/ru/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -19,62 +19,101 @@ msgstr ""
...
 
-#: templates/django_extensions/widgets/foreignkey_searchinput.html:4
-msgid "Lookup"
-msgstr "Поиск"
+#: validators.py:14
+msgid "Control Characters like new lines or tabs are not allowed."
+msgstr "Управляющие символы, такие как символ новой строки и символ табуляции недопустимы."
+
+#: validators.py:48
+msgid "Leading and Trailing whitespace is not allowed."
```
Comment (beaulirl) [inline django_extensions/locale/ru/LC_MESSAGES/django.po]: are

File: django_extensions/locale/ru/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -19,62 +19,101 @@ msgstr ""
...
+msgstr "Недопустимая длина, должно быть %(length)d символов."
+
+#: validators.py:76
+#, python-format
+msgid "Ensure that there are more then %(min)s characters."
+msgstr "Убедитесь, что длина строки больше %(min)s символов."
+
+#: validators.py:77
+#, python-format
+msgid "Ensure that there are no more then %(max)s characters."
```
Comment (beaulirl) [inline django_extensions/locale/ru/LC_MESSAGES/django.po]: than

File: django_extensions/locale/ru/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -19,62 +19,101 @@ msgstr ""
...
+msgid "Leading and Trailing whitespace is not allowed."
+msgstr "Пробел в начале или в конце недопустим."
+
+#: validators.py:74
+msgid "Only a hex string is allowed."
+msgstr "Допустимо использование только шестнадцатеричных строк."
+
+#: validators.py:75
+#, python-format
+msgid "Invalid length must be %(length)d characters."
```
Comment (beaulirl) [inline django_extensions/locale/ru/LC_MESSAGES/django.po]: Invalid length. Must be n characters.

File: django_extensions/locale/ru/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -19,62 +19,101 @@ msgstr ""
...
+msgstr "Недопустимая длина, должно быть %(length)d символов."
+
+#: validators.py:76
+#, python-format
+msgid "Ensure that there are more then %(min)s characters."
```
Comment (beaulirl) [inline django_extensions/locale/ru/LC_MESSAGES/django.po]: than

File: django_extensions/locale/ru/LC_MESSAGES/django.po (hunk 2 above)
Comment (chexex) [reply django_extensions/locale/ru/LC_MESSAGES/django.po]: Done

File: django_extensions/locale/ru/LC_MESSAGES/django.po (hunk 4 above)
Comment (chexex) [reply django_extensions/locale/ru/LC_MESSAGES/django.po]: Done

File: django_extensions/locale/ru/LC_MESSAGES/django.po (hunk 5 above)
Comment (chexex) [reply django_extensions/locale/ru/LC_MESSAGES/django.po]: Done

File: django_extensions/locale/ru/LC_MESSAGES/django.po (hunk 3 above)
Comment (chexex) [reply django_extensions/locale/ru/LC_MESSAGES/django.po]: Done

**Comment by trbs** [comment]
Thanks !

**Pull request merged**